pkg/objectio/column.go

```diff
@@ -0,0 +1,168 @@
+// Copyright 2021 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objectio
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"github.com/matrixorigin/matrixone/pkg/fileservice"
+	"github.com/matrixorigin/matrixone/pkg/vm/engine/tae/index"
+)
+
+type ColumnBlock struct {
+	meta  *ColumnMeta
+	block *Block
```

Comment:
This is kind of strange.   The Block can hold multiple columns, is that correct?   Now the ColumnBlock hold only one column, but holds a block.

Again, is there a reason to use pointer instead of just member?

Reply:
This block is actually the parent of ColumnBlock. The original intention is to use the “object” member in the block. Now it has been fixed.